### PR TITLE
Feature: Add Success Window

### PR DESCRIPTION
**Pull request by nGoline**
Adding a Success Window to guide the user through the pour and control the sensors/actuators

### CODE REVIEW

File: src/BitChopp.Main/ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -41,14 +46,27 @@ private async void OnSwitchSelected(SwitchCommandObject swObj)
...
+        var qrWindow = new QRCodeWindow(DeviceId, swObj.Switch.Pin, swObj.Switch.Lnurl, _configService)
+        {
+            Topmost = true // Make the window always on top
+        };
+        await qrWindow.ShowDialog(swObj.Window);
+
+        // if (qrWindow.WebSocketResult == "Paid")
+        // {
+        await ShowSuccessWindow(swObj); // replace with actual value
+        // }
```
Comment (lorenzolfm) [inline src/BitChopp.Main/ViewModels/MainViewModel.cs]: should you uncomment the if?

**Comment by johnnyasantoss** [comment]
Needs a rebase

File: src/BitChopp.Main/ViewModels/MainViewModel.cs (hunk 1 above)
Comment (nGoline) [reply src/BitChopp.Main/ViewModels/MainViewModel.cs]: Just testing for now. When we have the proper way of testing without paying I'll remove it.

**Review by jaonoctus — COMMENTED** [review]
nvm: missing "ml/L" in the end of the product name on lnbits on my side.

~after generating an invoice, if I click on "Cancelar" button, I got this error:~

```
Unhandled exception. System.InvalidOperationException: No volume information found in the string.
   at BitChopp.Main.ViewModels.MainViewModel.ExtractVolume(String text) in /home/jaonoctus/code/casa21/BitChopp/src/BitChopp.Main/ViewModels/MainViewModel.cs:line 116
   at BitChopp.Main.ViewModels.MainViewModel.ShowSuccessWindow(SwitchCommandObject swObj) in /home/jaonoctus/code/casa21/BitChopp/src/BitChopp.Main/ViewModels/MainViewModel.cs:line 66
   at BitChopp.Main.ViewModels.MainViewModel.OnSwitchSelected(SwitchCommandObject swObj) in /home/jaonoctus/code/casa21/BitChopp/src/BitChopp.Main/ViewModels/MainViewModel.cs:line 60
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state)
   at Avalonia.Threading.SendOrPostCallbackDispatcherOperation.InvokeCore()
   at Avalonia.Threading.DispatcherOperation.Execute()
   at Avalonia.Threading.Dispatcher.ExecuteJob(DispatcherOperation job)
   at Avalonia.Threading.Dispatcher.ExecuteJobsCore(Boolean fromExplicitBackgroundProcessingCallback)
   at Avalonia.Threading.Dispatcher.Signaled()
   at Avalonia.X11.X11PlatformThreading.CheckSignaled()
   at Avalonia.X11.X11PlatformThreading.RunLoop(CancellationToken cancellationToken)
   at Avalonia.Threading.DispatcherFrame.Run(IControlledDispatcherImpl impl)
   at Avalonia.Threading.Dispatcher.PushFrame(DispatcherFrame frame)
   at Avalonia.Threading.Dispatcher.MainLoop(CancellationToken cancellationToken)
   at Avalonia.Controls.ApplicationLifetimes.ClassicDesktopStyleApplicationLifetime.Start(String[] args)
   at Avalonia.ClassicDesktopStyleApplicationLifetimeExtensions.StartWithClassicDesktopLifetime(AppBuilder builder, String[] args, Action`1 lifetimeBuilder)
   at BitChopp.Main.Program.Main(String[] args) in /home/jaonoctus/code/casa21/BitChopp/src/BitChopp.Main/Program.cs:line 11
```

**Review by jaonoctus — COMMENTED** [review]
And after paying an invoice, the success windows don't show up

File: src/BitChopp.Main/ViewModels/MainViewModel.cs (hunk 1 above)
Comment (jaonoctus) [reply src/BitChopp.Main/ViewModels/MainViewModel.cs]: @nGoline Maybe include an env var to skip the payment request or something like that?

`if (qrWindow.WebSocketResult == "Paid" || isDebugMode)`

**Pull request merged**